src/components/LogIn/LogIn.js

```diff
@@ -0,0 +1,124 @@
+import React, { useState } from 'react';
+import SVG from '../../assets/login.svg';
+import {
+	Heading,
+	Flex,
+	VStack,
+	Image,
+	SimpleGrid,
+	GridItem,
+	FormControl,
+	FormLabel,
+	Input,
+	Text,
+	Button,
+} from '@chakra-ui/react';
+
+import { Link } from 'react-router-dom';
+import axios from 'axios';
+import * as Yup from 'yup';
+
+export default function LogIn() {
+	const [email, setEmail] = useState('');
+	const [password, setPassword] = useState('');
+	const [error, setError] = useState(false); // if any input validation error occur
+
+	// input validation schema
+	const logInSchema = Yup.object().shape({
+		email: Yup.string().email().required(),
+		password: Yup.string().min(8).required(),
+	});
+
+	const handleClick = async (e) => {
+		e.preventDefault();
+		setError(false); // resetting the error messgae
+		const isValid = await logInSchema.isValid({
+			email: email,
+			password: password,
+		});
+
+		if (isValid) {
+			const URI = 'http://localhost:5000/login'; // URI is subject to change
```

Comment:
api path will be 'http://localhost:5000/api/auth/signin' on backend

Reply:
done